frontend/server/src/main/java/org/pytorch/serve/http/InferenceRequestHandler.java

```diff
@@ -179,7 +179,9 @@ private void predict(
         Job job = new Job(ctx, modelName, modelVersion, WorkerCommands.PREDICT, input);
         if (!ModelManager.getInstance().addJob(job)) {
             throw new ServiceUnavailableException(
-                    "No worker is available to serve request: " + modelName);
+                    "No worker is available to serve request: "
```

Comment:
Add model version as well in response.

Reply:
Can we say here 'Model {} has no worker to serve inference request'?